articles/server-apis/rails/authenticate.md

```diff
@@ -113,7 +113,7 @@ You can now make requests against your secure API by providing the Authorization
 
 ### 5. You're done!
 
-Now you have both your FrontEnd and Backend configured to use Auth0. Congrats, you're awesome!
+Now you have both, your FrontEnd and Backend configured to use Auth0. Congrats, you're awesome!
```

Comment:
I dont think this is right. The comma isnt needed. Also it should be "frontend" not "FrontEnd" and "backend", no need to capitalize.